packages/xod-client/src/editor/actionTypes.js

```diff
@@ -36,3 +36,10 @@ export const START_DRAGGING_PATCH = 'START_DRAGGING_PATCH';
 export const INSTALL_LIBRARIES_BEGIN = 'INSTALL_LIBRARIES_BEGIN';
 export const INSTALL_LIBRARIES_COMPLETE = 'INSTALL_LIBRARIES_COMPLETE';
 export const INSTALL_LIBRARIES_FAILED = 'INSTALL_LIBRARIES_FAILED';
+
+export const SET_ALL_PANEL_SETTINGS = 'SET_PANEL_SETTINGS';
```

Comment:
Maybe I overcomplicate, but it seems bulky. What’s the *fact* “set all panel settings” mean… Well, if one thinks a second about it he can guess. But if I did the same, I’d call “all panel settings” a “sidebar layout”. I.e. it would be `SET_SIDEBAR_LAYOUT`. Not sure it makes much more sense. I leave it up to you.